### PR TITLE
fix: boolean logic

### LINKED ISSUE #93: No resources returned in v1.3.4
**Describe the bug**
No resources are being returned in the latest version of the krew plugin.

**To Reproduce**
1. kubectl get-all -n bar
2. See error:
`DEBU[0000] Set log-level to debug
DEBU[0001] Resources to fetch: []
DEBU[0001] Initial fetchResourcesBulk done (0s)
DEBU[0001] Fetch resources incrementally
DEBU[0001] Requests done (elapsed 0s)
WARN[0001] No resources found, are you authorized? Try to narrow the scope with --namespace.
FATA[0001] no resources found`

**Expected behavior**
I expected too see the same resources I could in v1.3.2

**Screenshots**
image

**Desktop (please complete the following information):**
 - output of `ketall version --full`:
> ketall:     v1.3.4
> platform:   darwin/amd64
> git commit: 071f5c7ec0f520ab9be82529c08e83fc726b4910
> build date: 2020-09-04T23:14:49Z
> go version: go1.15
> compiler:   gc
 - OS: MacOS
 - Browser: Firefox
 - Version 1.3.4

**Additional context**
Exact same command and kubectl version worked fine for version 1.3.2, seems to be a bug.

### PR DESCRIPTION
**Pull request by busser**
Fix #93

### CODE REVIEW

File: internal/client/client_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package client
+
+import "testing"
+
+func TestGetResourceScope(t *testing.T) {
+	tests := map[string]struct {
+		scope         string
+		wantCluster   bool
+		wantNamespace bool
+		wantErr       bool
```
Comment (corneliusweig) [inline internal/client/client_test.go:10]: No need for change, just a thought: If you introduce a struct like this
```go
type result struct {
  cluster, namespace, err bool
}
```

you can do all three comparisons at once like so:
```go
if diff := cmp.Diff(test.want, got); diff != "" {
  t.Errorf("getResourceScope(..) returned unexpected result (-want,+got)\n%s", diff)
}
```

**Review by corneliusweig — COMMENTED** [review]
:man_facepalming: This clearly shows why tests pay off!
Thanks for stepping up and fixing this!

Before I can merge, I need to ask you though to signoff your commit (see [here](https://github.com/corneliusweig/ketall/blob/master/CONTRIBUTING.md#pull-request-checklist)).

Fix #93

File: internal/client/client_test.go (hunk 1 above)
Comment (busser) [reply internal/client/client_test.go:10]: Neat! I'll keep that in mind for next time

**Comment by busser** [comment]
@corneliusweig Commits are signed off 👍

**Pull request merged**